example/property-overrides.yml

```diff
@@ -33,11 +33,16 @@ service_broker_properties:
   username: username
   password: password
   http_request_timeout: 5000
-  catalog:
-    service_name: autoscaler
-    service_id: autoscaler-guid
-    service_plan_name: autoscaler-free-plan
-    service_plan_id: autoscaler-free-plan-id
+  catalog: 
+    services:
+    - id: autoscaler-guid
+      name: autoscaler
+      description: Automatically increase or decrease the number of application instances based on a policy you define.
+      bindable: true
+      plans:
+      - id: autoscaler-free-plan-id
```

Comment:
In this change we added  "-"  for two properties, no where else in this yml, this convention is used. Raising just from consistency point of view.